readme.md

```diff
@@ -1,6 +1,6 @@
 # Lumar Protect
 
-[<img src="https://www.deepcrawl.com/wp-content/themes/deepcrawl/images/logo-footer.svg">](https://www.deepcrawl.com/)
+[<img src="images/lumar-logo.svg">](https://www.lumar.io/platform/protect)
```

Comment:
The new logo renders quite big in the readme.md compared to the old one, should we adjust the size?

Reply:
resized to same size as previous logo